### ISSUE TITLE
Question about  accuracy and evaluation setting

### ISSUE THREAD

**Issue by Gardlin** [hustvl/HAIS#17]
Hello, I follow your code  to evaluate on validation set ，but  it goes wrong  on the 17-th instance iter  with  " wrong number of line in mask#0 "  but I'm pretty sure  I follow your code to  run the `python prepare_data_inst_gttxt.py`  ,when I  replace the 17-th scene name in  the  **scannetv2_val.txt**   to any other scene_name  , it works well.
Besides , when i replace the 17-th scene_name with any other scene, the accuracy seems  lower than the   **mAP/mAP50/mAP25 is 44.1/64.4/75.7.**  provided in your github.
I  am wondering what causes this.

**Comment by Gardlin** [comment]
one more ,  I  found  the setting   ` using_set_aggr_in_training: False`  in the  config file  is  False  ,I  am doubting whether should I  set  it  True

**Comment by outsidercsy** [comment]
Hi. We haven't encountered this problem. Perhaps this bug is relative to data. Please try to re-download the annotation files and then run "python prepare_data_inst_gttxt.py" again.  
As for 'using_set_aggr_in_training', both True of False are OK. But disabling set aggr. in training makes the convergence more stable.

**Comment by Gardlin** [comment]
When  I  re-generate gt-files, the testing result of your pretrained models  now   correspondends to yours , but  the  testing result  of  **your  trained models for 500 epoch is  still lower for  2 or 3 point  in the mAP/mAP50/mAP25**  . I  want to  know how  can  I get the same  train  result with your provided pretrained model   on the valid  dataset.

**Comment by c6376315qqso** [comment]
I trained the model following the instruction for 500 epoch, the mAP50 is about 60, which is 4 point lower than the paper. Where is the problem ?
